pkg/apis/performanceprofile/v2/performanceprofile_types.go

```diff
@@ -176,16 +176,16 @@ type RealTimeKernel struct {
 // WorkloadHints defines the set of upper level flags for different type of workloads.
 type WorkloadHints struct {
 	// HighPowerConsumption defines if the node should be configured in high power consumption mode.
-	// The flag will affect the power consumption but will improve the CPUs latency.
+	// The flag will affect the power consumption but will improve the CPUs latency. Defaults to false.
 	// +optional
 	HighPowerConsumption *bool `json:"highPowerConsumption,omitempty"`
-	// RealTime defines if the node should be configured for the real time workload.
+	// RealTime defines if the node should be configured for the real time workload. Defaults to true.
 	// +default=true
 	// +optional
 	RealTime *bool `json:"realTime,omitempty"`
 	// +optional
 	// PerPodPowerManagement defines if the node should be configured in per pod power management.
-	// PerPodPowerManagement and HighPowerConsumption hints can not be enabled together.
+	// PerPodPowerManagement and HighPowerConsumption hints can not be enabled together. Defaults to false.
```

Comment:
I'm not blocking for this, but I'm not completely sold about describing the default value in the CRD. On one hand, defaults are unlikely to change. On the other hand, I don't think this balongs into the CRD, which defines the purpose of the fields. Values should be ideally learned querying the objects (status) not the CRD.

Reply:
Yeah, I absolutely agree in most other circumstances. Unfortunately, the default values between the 3 fields are different with realTime being true, the other 2 being false. And the oc adm explain just doesn't show the default values to users ... and neither does our documentation which I think is largely derived from the oc explain output. I think this is one of the very, very rare cases where it makes sense in order to avoid user confusion, even though it is ugly